frappe/public/js/frappe/form/controls/currency.js

```diff
@@ -1,22 +1,21 @@
 frappe.ui.form.ControlCurrency = frappe.ui.form.ControlFloat.extend({
 	eval_expression: function(value) {
-		if (typeof value === 'string' 
-			&& value.match(/^[0-9+-/* ]+$/)) {
-			if (value.includes(',')) {
-				// paresFloat('1,44,000') returns 1.0
-				// 1,44,000 are being passed when we paste rows from excel sheet to a table)
+		// Seperators are based on countries
+		const { group_sep } = get_number_format_info(this.get_number_format());
```

Comment:
I guess [this function](https://github.com/frappe/frappe/blob/develop/frappe/public/js/frappe/utils/number_format.js#L33) in number_format.js is doing the same thing.

You can use that function instead.

Please check and confirm.

Reply:
We can't use the UI at all without disabling the code and building again. Please consider this and act quickly.
Please test thoroughly and think about consequences next time...